src/Features/header/components/Cities/LoadingSpinner.tsx

```diff
@@ -0,0 +1,19 @@
+export const LoadingSpinner = () => {
```

Comment:
Cool! But we will use this spinner further, so put it in the `src/components` folder